kahuna/app/lib/KahunaConfig.scala

```diff
@@ -52,5 +52,9 @@ class KahunaConfig(resources: GridConfigResources) extends CommonConfig(resource
 
   val metadataTemplates: Seq[MetadataTemplate] = configuration.get[Seq[MetadataTemplate]]("metadata.templates")
 
+  //BBC custom warning text
+  val warningTextPrefix: String = configuration.getOptional[String]("warningText.prefix").getOrElse("")
```

Comment:
I think 'header' describes these better than 'prefix'

Reply:
plus, can we put the defaults in the getOrElse here? then the UI doesn't have to worry about it; should make the js code simpler

Reply:
makes sense